modules/apps/commerce/commerce-price-list-api/src/main/java/com/liferay/commerce/price/list/helper/CommercePriceEntryHelper.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.price.list.helper;
+
+import com.liferay.commerce.price.list.model.CommercePriceEntry;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.search.Indexable;
+import com.liferay.portal.kernel.search.IndexableType;
+
+/**
+ * @author Dante Wang
+ */
+public interface CommercePriceEntryHelper {
```

Comment:
@dantewang I do not like this solution, could you figure out a better way to address this issue?

Tina